ui/src/containers/Authentication/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import GitHubLogin from 'react-github-login';
+import { useHistory } from 'react-router-dom';
+import { Card, CardBody, CardHeader } from '@patternfly/react-core';
+import { GH_CLIENT_ID } from '../../config/constants';
+import { useMst } from '../../store/root';
+import { AuthCodeProps } from '../../store/auth';
+import { Icons } from '../../common/icons';
+import Icon from '../../components/Icon';
+import './Authentication.css';
+
+const Authentication: React.FC = () => {
+  const { user } = useMst();
+  const history = useHistory();
+
+  const refreshPage = () => {
+    history.push('/');
+    window.location.reload();
+  };
+
+  const onSuccess = (code: AuthCodeProps) => {
+    user.authenticate(code, refreshPage);
+  };
+
+  return (
+    <React.Fragment>
+      <Card className="hub-authentication-card__size">
+        <CardHeader className="hub-authentication-card__header">
+          <Icon id={Icons.Github} size="lg" label={'github'} />
+        </CardHeader>
+        <CardBody className="hub-authentication-card__body">
+          <GitHubLogin
+            clientId={GH_CLIENT_ID}
+            redirectUri=""
```

Comment:
If this is an empty string can we still need to add it ??

Reply:
yes,we need it